include/robot_localization/navsat_transform.hpp

```diff
@@ -57,7 +57,7 @@ class NavSatTransform
   /**
    * @brief Constructor
    */
-  NavSatTransform();
+  NavSatTransform(std::shared_ptr<rclcpp::Node> node_);
```

Comment:
And I'm also looking at the [cpp file](https://github.com/cra-ros-pkg/robot_localization/blob/3fed5670769ae397222d64ac3c6f129e2c358140/src/navsat_transform.cpp#L58) and maybe someone can school me on smart pointers a little here, but I don't actually see where the `node_` that is an input is given to the `node_` that is the member variable (or does C++11 auto-copy when you make them the same name O.o that seems bad)

Reply:
I just thought it should be seen as the private nodehandle in ROS1. I'll rename it. 

Should the private node class member be called node_ ?  

Regarding the pointer, you are right it is not explicitly assigned anywhere will fix it. Not sure what the default behavior would be, it compiled because its declared, but I guess running it would have resulted in an exception ? (I'm fairly new to C++)

Reply:
So member's have a trailing `_` in this code style, and inputs to members do not. So the constructor would look like `Name(type name)` and not `Name(type name_)` but the variable `name` is stored in is `name_` in the class